fsr.ino

```diff
@@ -26,6 +31,26 @@
   void ButtonRelease(uint8_t button_num) {
     Joystick.button(button_num, 0);
   }
+  void ButtonSend() {
+    Joystick.send_now();
+  }
+#elif defined(USE_ARDUINO_JOYSTICK_LIBRARY)
+  #include <Joystick.h>
+  // Create the Joystick
+  Joystick_ Joystick;
+  void ButtonStart() {
+    // Passing false disables autosend.
+    Joystick.begin(false);
+  }
+  void ButtonPress(uint8_t button_num) {
+    Joystick.pressButton(button_num - 1);
```

Comment:
Does this library allow `pressButton` to accept a `buttom_num` of 0? At least for Teensy it wouldn't let you so I wonder if we actually want the `- 1` here?

Reply:
You could leave the `- 1` out and it would just skip one button, there are 32 buttons most of which are unused anyway. But yes it does work.

Reply:
Right I get that I just meant that button 0 is not a thing for the teensy library (https://www.pjrc.com/teensy/td_joystick.html)

I think for consistency, let's make all the joysticks start at 1.

Reply:
Can do.